README.md

```diff
@@ -24,17 +23,52 @@ This Kubernetes-based version is built using the same approach as the [machine c
 a situation where your Kubernetes cluster and its ingress controllers have
 a fast connection to the Internet.
 
+For information about how to deploy, integrate, and manage this charm, see the Official [Content Cache Documentation](https://charmhub.io/content-cache-k8s/docs).
+
+## Get started
+
+To begin, refer to the [Content Cache tutorial](https://charmhub.io/content-cache-k8s/docs/tutorial-getting-started) for step-by-step instructions.
+
+### Basic operations
+
+The following actions are available for the charm:
+- report-visits-by-ip
+
+Tuning options include:
+- cache storage size
+- maximum request size to cache 
+- cache validity duration
+
+You can find more information about supported actions in [the Charmhub documentation](https://charmhub.io/content-cache-k8s/actions).
+
+## Integrations
+
+Content-cache is meant to serve as cache for another charm. You can use Hello-kubecon as an example:
```

Comment:
Hello-kubecon is old and not updated, so might be better to use anycharm here:
https://github.com/canonical/any-charm

Reply:
I think the `any-charm` might not be ideal for this situation, as it is oriented toward automated testing, which might make manual operation difficult, especially for beginners.

By the way, I believe the `ingress-proxy` has been renamed to `nginx-proxy` in the latest version of the `content-cache-k8e` charm. For `nginx-route`-like integrations, the easiest charm to use might be the `wordpress-k8s` charm. You can try deploying and integrating it (untested):

```
juju deploy wordpress-k8s
juju deploy mysql-k8s --trust
juju integrate wordpress-k8s mysql-k8s:database

juju integrate content-cache-k8s:nginx-proxy wordpress-k8s
```

Reply:
updated the README with wordpress charm